[12.0][FIX] web_responsive: Do not set overflow property on inline fields

ypapouin opened the pull request:
The responsive css does not exclude inline fields when it wants enabled ellipsis on titles and it leads to wrong align.

Before:
![image](https://user-images.githubusercontent.com/22446243/107046842-62e10900-67c7-11eb-85b6-2fd17888c427.png)

After:
![image](https://user-images.githubusercontent.com/22446243/107046809-56f54700-67c7-11eb-9eb6-db6837e1f084.png)

Tardo commented on web_responsive/static/src/css/web_responsive.scss:
```diff
@@ -397,11 +397,13 @@ html .o_web_client .o_main .o_main_content {
+                }
             }
             span:active {
                 white-space: normal;
```
```suggestion
                white-space: normal !important;
```

Tardo reviewed (changes requested):
Thanks... it's confused how in some views the oe_title looks like a head section more than a simple title section.

Please, apply this other fix to ensure that the title is displayed while the user holds the mouse click.

pedrobaeza commented:
Please squash commits. Aren't any other way to avoid the `!important`?

Tardo commented on web_responsive/static/src/css/web_responsive.scss:
```diff
@@ -397,14 +397,16 @@ html .o_web_client .o_main .o_main_content {
+                    width: initial;
+                }
             }
             span:active {
```
```suggestion
```

Tardo commented on web_responsive/static/src/css/web_responsive.scss:
```diff
@@ -397,14 +397,16 @@ html .o_web_client .o_main .o_main_content {
             }
             span:active {
-                white-space: normal;
+                white-space: normal !important;
```
```suggestion
```

Tardo commented on web_responsive/static/src/css/web_responsive.scss:
```diff
@@ -397,14 +397,16 @@ html .o_web_client .o_main .o_main_content {
             span:active {
-                white-space: normal;
+                white-space: normal !important;
             }
```
```suggestion
```

Tardo commented on web_responsive/static/src/css/web_responsive.scss:
```diff
@@ -397,14 +397,16 @@ html .o_web_client .o_main .o_main_content {
-                white-space: nowrap;
-                overflow: hidden;
-                width: initial;
+                &:not(.oe_inline) {
+                    max-width: 100%;
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                    overflow: hidden;
+                    width: initial;
+                }
```
```suggestion
                &:not(.oe_inline) {
                    max-width: 100%;
                    text-overflow: ellipsis;
                    white-space: nowrap;
                    overflow: hidden;
                    width: initial;
                    
                    &:active {
                        white-space: normal;
                    }
                }
```

Tardo reviewed (commented):
To avoid "!important" usage.

ypapouin committed:
[FIX] web_responsive: Do not set overflow property on inline fields
Co-authored-by: Alexandre Díaz <alexandre.diaz@tecnativa.com>
Ensure that the title is displayed while the user holds the mouse click (while selecting content)

pedrobaeza commented:
Please squash commits together.

Tardo reviewed (approved):
LGTM. Thanks.

pedrobaeza commented:
/ocabot merge patch

is this needed in 13.0?

ypapouin commented:
> 
> 
> /ocabot merge patch
> 
> is this needed in 13.0?

Yes, tested on runbot.
Is there an automated way from github to cherry pick these commits and create a branch+PR targeting 13.0 ?

pedrobaeza commented:
I'm afraid not, but you can use `git cherry-pick` locally for applying the patch on 13.0 branch, and then propose the PR. If you have problems, let me know.
